spec/requests/api/v1/users/index_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe 'GET /users' do
+  describe 'when the records exist' do
+    it 'returns all users in json format' do
+      json_response = File.read('spec/fixtures/DO_NOT_DELETE/users.json')
+      stub_request(:get, "http://localhost:3000/api/v1/users").to_return(status: 200, body: json_response)
```

Comment:
You shouldn't need to stub the request for a request spec. We actually want rspec to make a real request to our local application so we can test what it responds. If you're getting errors, it probably means you need to set up so test data to make sure there are users and trails in your test db before this test (Similar to what you did in the test above!)